src/main/resources/reference.conf

```diff
@@ -1354,6 +1354,106 @@ resourceTypes = {
     }
     reuseIds = false
   }
+  mct_mapping = {
+    actionPatterns = {
+        delete = {
+            description = "delete this resource"
+        }
+        read_policies = {
+            description = "view all policies and policy details for this resource"
+          }
+          set_parent = {
```

Comment:
Will this resource need a parent/child relationship?  If not, you can just get rid of this action

Reply:
at least initially we don't need it, thanks